packages/app/src/app/pages/Dashboard/Components/Sandbox/SandboxCard.tsx

```diff
@@ -73,9 +73,11 @@ const SandboxTitle: React.FC<SandboxTitleProps> = React.memo(
           </form>
         ) : (
           <Stack gap={3} align="flex-start" css={{ overflow: 'hidden' }}>
-            <Element css={{ flexShrink: 0 }}>
-              <TemplateIcon width="16" height="16" />
-            </Element>
+            {TemplateIcon && (
+              <Element css={{ flexShrink: 0 }}>
```

Comment:
looking at this feels so strange, I guess this is what tailwind will do to you long term :)))